### ISSUE TITLE
Hide administration pages

### ISSUE THREAD

**Issue by milescalabresi** [learning-unlimited/ESP-Website#3497]
Before Will and Katherine made /manage interfaces for the modules, the /admin pages interface was necessary for anyone who wanted to modify modules, costs, and many other program configurations. Now it is not, and gives webmins the power to mess things up. I think we should remove the "Administration pages" from the admin toolbar on the left and add a warning (landing page, header, or popup) that the user should probably go to /manage/current/main instead unless they really know what they're doing.

**Comment by willgearty** [comment]
I'm hesitant to remove it entirely because I know people more familiar with the website are used to it being there...but at the same time, it does seem to cause more confusion for less experienced folks.

At the very least, we should always include the real link for dev servers.

**Comment by milescalabresi** [comment]
Yes agreed for dev servers. Per our conversation, these items would benefit from having safer "front-end" UIs. We can pare down this list and/or discuss more as needed, and these definitely vary in priority
- Cal
  - events - can't change description of teacher trainings
- Dbmail
  - dbreceipts
  - plain redirects
- Programs
  - class categories
  - class flag types
  - phase zero records (add a student to the lottery)
  - program module objs (high priority)
  - registration types (low priority)
- Redirects
- Users
  - Grade Change requests (but you get an email); maybe it should be on the userview page
  - Permissions (for individual users)
- Utils
  - printers (low priority)
  - Template Overrides for schedules
